src/Endpoints/CustomHostnames.php

```diff
@@ -30,27 +30,52 @@ public function __construct(Adapter $adapter)
      * @param string $hostname
      * @param string $sslMethod
      * @param string $sslType
-     * @param array $sslSettings
+     * @param array  $sslSettings
      * @param string $customOriginServer
-     * @param bool $wildcard
+     * @param bool   $wildcard
+     * @param string $bundleMethod
+     * @param string $customKey
+     * @param string $customCertificate
      * @return \stdClass
      */
-    public function addHostname(string $zoneID, string $hostname, string $sslMethod = 'http', string $sslType = 'dv', array $sslSettings = [], string $customOriginServer = '', bool $wildcard = false): \stdClass
-    {
+    public function addHostname(
+        string $zoneID,
+        string $hostname,
+        string $sslMethod = 'http',
+        string $sslType = 'dv',
+        array $sslSettings = [],
+        string $customOriginServer = '',
+        bool $wildcard = false,
+        string $bundleMethod = '',
+        string $customKey = '',
+        string $customCertificate = ''
+    ): \stdClass {
         $options = [
             'hostname' => $hostname,
             'ssl' => [
                 'method' => $sslMethod,
                 'type' => $sslType,
-                'settings' => $sslSettings
+                'settings' => $sslSettings,
+                'wildcard' => $wildcard,
             ],
-            'wildcard' => $wildcard,
         ];
 
         if (!empty($customOriginServer)) {
             $options['custom_origin_server'] = $customOriginServer;
         }
 
+        if (empty($bundleMethod) === false) {
```

Comment:
wouldn't `!empty` suffice here?

Reply:
It would, I guess I used the strict typing out of habit as I find it easier to read when skim reading the code. I've updated these for you

Reply:
It would, I guess I used the strict typing out of habit as I find it easier to read when skim reading the code. I've updated these for you